test/model/compose_test.dart

```diff
@@ -316,6 +316,22 @@ hello
       check(wildcardMention(WildcardMentionOption.topic, store: store()))
         .equals('@**topic**');
     });
+
+    group('user group', () {
+      final userGroup = eg.userGroup(id: 123, name: 'Group Name');
```

Comment:
nit: ID isn't relevant to understanding these tests, so can omit

```suggestion
      final userGroup = eg.userGroup(name: 'Group Name');
```

(contrasts with the user case, where it can appear in the output)